Makefile

```diff
@@ -42,7 +49,8 @@ build-dashboards:
 	@echo "Building dashboards"
 	@$(ENVVARS) $(GOCMD) run $(GOMAIN) --output-dir="./examples/dashboards/operator" --output="operator" --project="perses-dev" --datasource="prometheus-datasource"
 	@$(ENVVARS) $(GOCMD) run $(GOMAIN) --output-dir="./examples/dashboards/perses" --output="yaml" --project="perses-dev" --datasource="prometheus-datasource"
-	@$(ENVVARS) $(GOCMD) run $(GOMAIN) --output-dir="./examples/dashboards/json/operator/" --output="operator-json" --project="perses-dev" --datasource="prometheus-datasource"
+	@$(ENVVARS) $(GOCMD) run $(GOMAIN) --output-dir="./jsonnet/dashboards/operator/" --output="operator-json" --project="perses-dev" --datasource="prometheus-datasource"
+	$(MAKE) jsonnet-resources
```

Comment:
shouldn't we have a new target that combines `build-dashboards` and `jsonnet-resources` (or move the existing `build-dashboards` to a new intermediate target)?

Reply:
Yeah I was keeping all of them the same as a one-shot generate target, but you're right, let me make it cleaner. We already have a generate target

Reply:
Addressed